### DISCUSSION TITLE
making a ytpmv from a audio file instead of midi

### DISCUSSION THREAD

**Discussion by insertnameuserhere** [otomad/OtomadHelper#81] [Show and tell]
[preview](https://www.youtube.com/watch?v=i9N_bXGhjdE&t)
(keep the midi format if your the owner & if your reading this)

**Comment by otomad** [comment]
Duplicate #67

**Comment by otomad** [comment]
This might be what you want:
https://colab.research.google.com/github/magenta/mt3/blob/main/mt3/colab/music_transcription_with_transformers.ipynb

If you know how to code, here's the source code for the project:
https://github.com/magenta/mt3

**Comment by Evauation** [comment]
Asked for the same thing again 6 months later. Like Otomad Helper is a tool, not an all-in-one solution for every need.
